data/mode.txt

```diff
@@ -1 +1 @@
-extended
```

Comment:
Revert this when you can

Reply:
Yeah will do, it keeps doing that when I force the round to be traitor

Reply:
Changed now that most reviews are done